prompt_toolkit/output/color_depth.py

```diff
@@ -46,9 +50,12 @@ def default(cls, term: Optional[str] = None) -> "ColorDepth":
         override this outcome. This is a way to enforce a certain color depth
         in all prompt_toolkit applications.
 
-        If no `term` parameter is given, we use the $TERM environment variable.
+        If no `output` parameter is given or it does not provide a corresponding
+        `term`, we use the $TERM environment variable.
         """
         # Take `TERM` value from environment variable if nothing was passed.
+        term = getattr(output, "term", None)
```

Comment:
Like the comment below, this code where we check "term" can probably move to ``Vt100Output.get_default_color_depth()``.